docs/guide/best_practices.md

```diff
@@ -151,3 +151,12 @@ ezpublish:
 This setting **does not change Symfony behavior** regarding redirection after login. If set, it will only substitute the value set for `default_target_path`. It is therefore still possible to specify a custom target path using a dedicated form parameter.
 
 **Order of precedence is not modified.**
+
+### Copy subtree limit
+
+Copying large subtrees can cause performance issues, so you can limit the number of Content items
+that can be copied at once using `ezsystems.platformui.application_config.copy_subtree.limit`
+in `parameters.yml`.
+
+The default value is `100`. You can set it to `-1` for no limit,
+or to `0` to completely disable copying subtrees.
```

Comment:
Why is there a break line?

Reply:
So the lines aren't too long and hard to review. Also, per our own guidelines: https://doc.ezplatform.com/en/1.12/community_resources/contributing/#writing-guidelines-short-version